src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSidebar/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { Box } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { MyCollectionArtworkSidebarMetadataFragmentContainer } from "./MyCollectionArtworkSidebarMetadata"
+import { MyCollectionArtworkSidebar_artwork } from "__generated__/MyCollectionArtworkSidebar_artwork.graphql"
+
+const MyCollectionArtworkSidebarContainer = Box
+
+interface MyCollectionArtworkSidebarProps {
+  artwork: MyCollectionArtworkSidebar_artwork
+}
+
+export const MyCollectionArtworkSidebar: React.FC<MyCollectionArtworkSidebarProps> = ({
+  artwork,
+}) => {
+  return (
+    <MyCollectionArtworkSidebarContainer>
+      <MyCollectionArtworkSidebarMetadataFragmentContainer artwork={artwork} />
+    </MyCollectionArtworkSidebarContainer>
```

Comment:
**thought:** is there a reason why you renamed the `Box` to `MyCollectionArtworkSidebarContainer`? maybe just anticipating the upcoming work?

Reply:
Yes just for the upcoming work, but I don't mind renaming it to Box :)
And I forgot to add the placeholder 😅